components/ProfileCard/accountSettingsValidation.ts

```diff
@@ -0,0 +1,12 @@
+import { object, string, ref } from 'yup';
+
+export const accountSettingsSchema = object({
+  newEmail: string().email('Invalid email format').required('Email is required'),
```

Comment:
New email is not required. How you have it now means that every time we want to change our password, we also have to change our email